.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+# TODO: Add ruff + ruff format for nice linting and consistency
```

Comment:
Add pre-commit to make it easier to contribute to the repo -- TODO: standardize how to install this? Should we add a Makefile?